### PR TITLE
Playtest Planning

### PR DESCRIPTION
**Pull request by RavenholmZombie**

### CODE REVIEW

**Review by CSX8600 — CHANGES_REQUESTED** [review]
Unfortunately it seems a lot of this information is already outdated.

Easy come easy go 🤷‍♂️

**Comment by RavenholmZombie** [comment]
> Unfortunately it seems a lot of this information is already outdated.
> 
> Easy come easy go 🤷‍♂️

Can you please specify what needs to be updated? Thanks.

File: about.html [hunk 1]
```diff
@@ -335,6 +329,72 @@ <h6 class="card-subtitle mbr-fonts-style mb-3 display-4">
...
+                <div class="row align-items-center">
+                    <div class="col-12 col-md-4">
+                        <div class="image-wrapper">
+                            <img src="assets/images/default-816x816.webp" alt="Mobirise Website Builder">
+                        </div>
+                    </div>
+                    <div class="col-12 col-md">
+                        <div class="card-box">
+                            <h5 class="card-title mbr-fonts-style m-0 mb-3 display-5">
+                                <strong>Hawkins</strong></h5>
```
Comment (CSX8600) [inline about.html]: No longer on the team

File: about.html [hunk 2]
```diff
@@ -335,6 +329,72 @@ <h6 class="card-subtitle mbr-fonts-style mb-3 display-4">
...
+                <div class="row align-items-center">
+                    <div class="col-12 col-md-4">
+                        <div class="image-wrapper">
+                            <img src="assets/images/default-816x816.webp" alt="Mobirise Website Builder">
+                        </div>
+                    </div>
+                    <div class="col-12 col-md">
+                        <div class="card-box">
+                            <h5 class="card-title mbr-fonts-style m-0 mb-3 display-5">
+                                <strong>Camtrack626</strong></h5>
```
Comment (CSX8600) [inline about.html]: No longer on the team

File: about.html [hunk 3]
```diff
@@ -268,6 +230,38 @@ <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-2">
     </div>
 </section>
```
Comment (CSX8600) [inline about.html]: Since you're re-working this About section - should Black Alert be on here?

File: about.html [hunk 4]
```diff
@@ -268,6 +230,38 @@ <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-2">
...
+                <div class="row align-items-center">
+                    <div class="col-12 col-md-4">
+                        <div class="image-wrapper">
+                            <img src="assets/images/picture-day-585x556.webp" alt="Mobirise Website Builder">
+                        </div>
+                    </div>
+                    <div class="col-12 col-md">
+                        <div class="card-box">
+                            <h5 class="card-title mbr-fonts-style m-0 mb-3 display-5">
+                                <strong>Probablyry</strong></h5>
```
Comment (CSX8600) [inline about.html]: Double check probablyry is still active - showing last activity of 7/17, which is beyond our 1-month activity window.

**Review by CSX8600 — COMMENTED** [review]
You have this PR labeled "Playtest Planning" but didn't make any content changes regarding the playtest. Did you forget to commit?

File: about.html (hunk 3 above)
Comment (RavenholmZombie) [reply about.html]: Black Alert only has the Citizen role. It is not known if he is supposed to be a member of server staff or not.

**Review by CSX8600 — APPROVED** [review]

**Pull request merged**